src/uproot/containers.py

```diff
@@ -864,7 +864,9 @@ class AsVectorLike(AsContainer):
 
     def __init__(self, header, values):
         self.header = header
-        if isinstance(values, AsContainer):
+        if isinstance(self, uproot.containers.AsBitSet):
+            self._items = numpy.dtype(numpy.bool_)
+        elif isinstance(values, AsContainer):
             self._items = values
```

Comment:
It would be nicer if this knowledge (`std::bitset` values are stored as bytes) was in the `AsBitSet` class somehow, but if this is the only way it differs from other vectors, then this is okay.